<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoulKIClass</class>
 <widget class="QMainWindow" name="MoulKIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoulKI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Vault</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeWidget" name="vaultTree">
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTabWidget" name="nodeDataArea">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="rawTab">
             <attribute name="title">
              <string>Node</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableWidget" name="nodeData">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="rowCount">
                 <number>32</number>
                </property>
                <property name="columnCount">
                 <number>2</number>
                </property>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string>Field</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Value</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="imageTab">
             <attribute name="title">
              <string>Image</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Title:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="imageNodeTitle"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="imageLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="saveImageButton">
                  <property name="text">
                   <string>Save Image</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="loadImageButton">
                  <property name="text">
                   <string>Load Image</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="textTab">
             <attribute name="title">
              <string>Text</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Title:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="textNodeTitle"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="textNodeEdit"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextEdit" name="chatPane"/>
          </item>
          <item>
           <widget class="QTreeWidget" name="playersTree">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="chatEntry"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAuth">
    <property name="title">
     <string>Auth</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuVault">
    <property name="title">
     <string>Vault</string>
    </property>
    <addaction name="actionSubscribe"/>
    <addaction name="actionSet_Active"/>
    <addaction name="actionFind_Node"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Vault"/>
    <addaction name="actionLoad_Vault"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionJoin_Age"/>
    <addaction name="actionLeave_Age"/>
   </widget>
   <addaction name="menuAuth"/>
   <addaction name="menuVault"/>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionCreate_Node">
   <property name="text">
    <string>Create Node</string>
   </property>
  </action>
  <action name="actionSubscribe">
   <property name="text">
    <string>Fetch Tree</string>
   </property>
  </action>
  <action name="actionSave_Vault">
   <property name="text">
    <string>Save Vault</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionSet_Active">
   <property name="text">
    <string>Set Active</string>
   </property>
  </action>
  <action name="actionJoin_Age">
   <property name="text">
    <string>Join Age</string>
   </property>
  </action>
  <action name="actionLeave_Age">
   <property name="text">
    <string>Leave Age</string>
   </property>
  </action>
  <action name="actionLoad_Vault">
   <property name="text">
    <string>Load Vault</string>
   </property>
  </action>
  <action name="actionFind_Node">
   <property name="text">
    <string>Find Node</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
